Dockerfile.cache-in-ci

```diff
@@ -5,6 +5,9 @@ USER 0
 RUN cd java-components/cache && mvn clean package -DskipTests
 
 FROM registry.access.redhat.com/ubi8/openjdk-17-runtime:1.13
+USER 0
+RUN mkdir /cache && chown 185 /cache
+USER 185
```

Comment:
ahh ... I bet this is the no longer need privileged change you are talking about

side bar, is there something special about user 185?  In case you were not aware, when the pod runs in openshift, the actual UID will not be 185.

Reply:
I would think this change would allow us to remove `runAsUse: 0` then in https://github.com/redhat-appstudio/jvm-build-service/blob/650846d2011d745b82c72cd59cf1e2398daeddfa/deploy/cache/base/deployment.yaml#L24

did you try that @stuartwdouglas ?

Reply:
User 185 comes from the base image, that file is only still there to allow infra deployments to work in the short term, as now the cache is workspace scoped.